iocp/data/patterns.ini

```diff
@@ -7,19 +7,19 @@ pattern:	\b(([a-z0-9\-]{2,}\[?\.\]?)+(abogado|ac|academy|accountants|active|acto
 defang:		True
 
 [IP]
-pattern:	\b(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})\b
+pattern:	\b((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$\b
 
 [Email]
-pattern:	\b([a-z][_a-z0-9-.]+@[a-z0-9-]+\.[a-z]+)\b
+pattern:	\b([a-zA-Z0-9_\-\.]+)@([a-zA-Z0-9_\-\.]+)\.([a-zA-Z]{2,5})$\b
 
 [MD5]
-pattern:	\b([a-f0-9]{32}|[A-F0-9]{32})\b
+pattern:	\b[a-fA-F0-9]{32}(:.+)?$\b
```

Comment:
ok but need to remove the $ at the end in order to match the MD5 in any part of the document and not only at the end of the line (example: a04e72503528dfc132c48e95fa3160ad|putty.exe)